.github/workflows/sync-wiki.yml

```
@@ -0,0 +1,194 @@
+        description: 'Force sync all wiki content'
+        required: false
+        default: 'false'
+        type: boolean
+
+jobs:
+  sync-wiki:
+    runs-on: ubuntu-latest
+    if: github.repository == 'hmislk/hmis'
+    
+    steps:
+    - name: Checkout main repository
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)